draft-ietf-httpbis-client-cert-field.md

```diff
@@ -323,14 +323,14 @@ from the TTRP). A TTRP MUST sanitize the incoming request before forwarding it
 on by removing or overwriting any existing instances of the fields. Otherwise,
 arbitrary clients can control the field values as seen and used by the backend
 server. It is important to note that neglecting to prevent field injection does
-not "fail safe" in that the nominal functionality will still work as expected
```

Comment:
I suspect this was correct before; "fail safe" is a verb, where "fail-safe" would be an adjective or used as a noun.